src/fetch/transfer_file.py

```diff
@@ -108,6 +108,7 @@ def auto_propose(
         category=Category.GENERAL,
     )
 
+    # payout_transfers_temp = construct_payouts(
     payout_transfers_temp = construct_payouts(
```

Comment:
This file would be untouched if...
```suggestion
    payout_transfers_temp = construct_payouts(
```

Reply:
Oh, forgot to delete that.